Tasks/VsTestV2/inputparser.ts

```diff
@@ -536,14 +536,25 @@ function getTestPlatformPath(inputDataContract : idc.InputDataContract) {
 }
 
 function getVSTestConsolePath(versionLowerLimit : string, versionUpperLimit : string): string {
-    const vswhereTool = tl.tool(path.join(__dirname, 'vswhere.exe'));
+    let vswhereTool = tl.tool(path.join(__dirname, 'vswhere.exe'));
 
     vswhereTool.line(`-version [${versionLowerLimit},${versionUpperLimit}) -latest -products * -requires Microsoft.VisualStudio.PackageGroup.TestTools.Core -property installationPath`);
     let vsPath = vswhereTool.execSync({ silent: true } as tr.IExecSyncOptions).stdout;
     vsPath = utils.Helper.trimString(vsPath);
-    tl.debug('Visual Studio 15.0 or higher installed path: ' + vsPath);
 
     if (!utils.Helper.isNullOrWhitespace(vsPath)) {
+        tl.debug('Visual Studio 15.0 or higher installed path: ' + vsPath);
+        return vsPath;
+    }
+
+    // Look for build tool installation if full VS not present
+    tl.debug('Attempting to find vstest.console from a visual studio build tools installation.');
+    vswhereTool = tl.tool(path.join(__dirname, 'vswhere.exe'));
+    vswhereTool.line(`-version [${versionLowerLimit},${versionUpperLimit}) -latest -products * -requires Microsoft.VisualStudio.Component.TestTools.BuildTools -property installationPath`);
+    vsPath = vswhereTool.execSync({ silent: true } as tr.IExecSyncOptions).stdout;
+    vsPath = utils.Helper.trimString(vsPath);
+    if (!utils.Helper.isNullOrWhitespace(vsPath)) {
+        tl.debug('Build tools installed path: ' + vsPath);
```

Comment:
can we please have this as an info message so that the user is aware of the SKU being chosen.. this is an important info IMO

Reply:
okay

Reply:
done